src/components/GraphCard/StackedGraph.tsx

```diff
@@ -36,18 +36,8 @@ type IStackedGraphProps = (
 
 const StackedGraph: FC<IStackedGraphProps> = ({ dataUsers, dataUserCategories}) => {
     //store
+    const currency = useAppSelector(state => state.persistedCurrencySlice.currency)
```

Comment:
useAppSelector<Here type of store>(state => state.persistedCurrencySlice.currency)